index.d.ts

```diff
@@ -0,0 +1,41 @@
+declare module 'react-swipeable' {
```

Comment:
digging around a bit it doesn't appear this `declare module` is needed?

Reply:
referencing types:
- https://github.com/react-spring/react-with-gesture/blob/master/index.d.ts
- https://github.com/FormidableLabs/nuka-carousel/blob/master/index.d.ts
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-redux/v6/index.d.ts

Reply:
Maybe, definitely seems possible without the module declaration.

It was my starting point as vscode suggested it:
![image](https://user-images.githubusercontent.com/10128574/54988521-8ec9bb80-4fae-11e9-80f3-a536bcb7f599.png)